fix.sh

```diff
@@ -136,6 +136,7 @@ ensure_bundle() {
   # update --conservative' flag works as expected - important when
```

Comment:
[bigfiles] This file, one of the 3 largest in the project, increased in size to 367 lines.  The total size of those files is now 743 lines (target: 696).  Is this file complex enough to refactor?